test/evergreen.yml

```diff
@@ -2107,6 +2107,32 @@ tasks:
             # Run Python testing for all tiered tests
             ${test_env_vars|} ${python_binary|python3} ../test/suite/run.py -j $(nproc) tiered
 
+  - name: tiered-test-small
+    tags: ["pull_request"]
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          posix_configure_flags: -DENABLE_STRICT=0 -DHAVE_DIAGNOSTIC=1 -DENABLE_S3=1 -DIMPORT_S3_SDK=external -DENABLE_PYTHON=1
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/cmake_build"
+          shell: bash
+          silent: true
+          script: |
+            set -o errexit
+            export AWS_ACCESS_KEY_ID=${aws_sdk_s3_ext_access_key}
+            export AWS_SECRET_ACCESS_KEY=${aws_sdk_s3_ext_secret_key}
+
+            # Only set verbose after having put the AWS access credentials into the environment.
+            set -o verbose
+
+            # Run S3 extension unit testing
+            ext/storage_sources/s3_store/test/run_s3_unit_tests
+            # Run Python testing for tiered06 
+            ${test_env_vars|} ${python_binary|python3} ../test/suite/run.py -j $(nproc) tiered06
```

Comment:
Just to confirm: since `tiered06` is part of the Python test suite, and will be executed by one of the `unit-test-bucket-xx` tasks. And the difference of this task is the invocation of the S3 extension on line 2131, right?

Reply:
The difference of this task is that we wanted to test a much smaller subset of the `tiered-storage-extensions-test` in pull-requests, so we are only invoking tiered06 out of all the tiered tests.

Reply:
Got it, thanks